src/core/Table/TableRowMemoized.tsx

```diff
@@ -58,8 +61,33 @@ const TableRow = <T extends Record<string, unknown>>(props: {
           );
         })}
       </div>
-      {row.isExpanded && subComponent && (
```

Comment:
This prop is a leftover and not needed?

Reply:
![image](https://user-images.githubusercontent.com/9084735/123801888-ae03d180-d8b8-11eb-94dd-0c7ce5f6ea55.png)

Because I set `unmountOnExit=true`, it will leave the DOM when `row.isExpanded` is false